CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 2.4.0
+* Add the macro version of `produce_or_load` to enable tagging with the calling source file and line.
+* Refactor `tag!` to avoid code duplication.
```

Comment:
Users don't have to know this :)

Reply:
Now that you say it, that makes sense. :-)